src/index.html

```diff
@@ -40,5 +40,6 @@
 <script src="javascript/app/ng-wig/ngWigToolbarProvider.js"></script>
 <script src="javascript/app/ng-wig/ngWigPluginDirective.js"></script>
 <script src="javascript/app/plugins/formats.ngWig.js"></script>
+<script src="javascript/app/plugins/forecolor.ngWig.js"></script>
```

Comment:
I thought you would include it in demo-plugin-colorpicker.html to show how easy is to work: switch off and on some of them. 

I think "formats" will also go to own demo file. It was just first plugin